corehq/apps/domainsync/management/commands/copy_domain.py

```diff
@@ -91,11 +92,23 @@ class Command(BaseCommand):
                          "copy. This must be false if running in a supervised process")
     )
 
+    def iter_source_dbs(self):
+        for sourcedb_name, sourcedb in self.source_couch.all_dbs_by_slug.items():
+            if sourcedb_name not in self.exclude_dbs:
+                print "In {} db".format(sourcedb_name or "the main")
+                yield sourcedb_name, sourcedb
+
     def handle(self, *args, **options):
         if len(args) not in [2, 3]:
             raise CommandError('Usage is copy_domain %s' % self.args)
-
-        sourcedb = Database(args[0])
+        self.exclude_dbs = (
+            # these have data we don't want to copy
+            'receiverwrapper', 'couchlog', 'auditcare', 'fluff-bihar', 'fluff-opm',
+            'fluff-mc', 'fluff-cvsu', 'mvp-indicators', 'm4change',
+            # todo: missing domain/docs, but probably want to add back
+            'meta',
```

Comment:
This is the only remaining issue. @esoergel I added a note to the cleanup section of [Domain Migration Notes](https://docs.google.com/document/d/1Rc8pYOHkW5QPOCOHqwpooSY3bW45qD8VAOhwCEXdP2o/edit#)